Khan/khan-dotfiles
# Dotfile fixes for intel

(Some of these improve M1 install too)

* Remove mac-setup-postgres.sh as it is a duplicate of mac-setup-postgres.py and does less at this point.
* Fix install-mac-homebrew.py to work on intel
* Fix mac-uninstall-brew.sh to work on intel
* Fix rebuild_virtualenv.sh to properly install py2 virtualenv
* Fix mac-setup-normal.sh to properly test ssh github access (No idea how this worked before)
* Fix setup.sh to re-install py2 virtualenv after py3 pipenv overrwrites the py2 virtualenv!

Issue: NA

Test plan:
* Verify mac-uninstall-homebrew.py works on both intel and M1
* Then verify 'make' works on intel and M1
* Ensure py2 version of virtualenv is installed (just head it)
* Validate ssh github access is tested properly when setup correctly and incorrectly.
